spec.bs

```diff
@@ -2800,6 +2803,143 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a `fetch()` call using the `{adAuctionHeaders: true}`

```

Comment:
`fetch()` should be `{{fetch()}}` I think? We want to link to https://fetch.spec.whatwg.org/#dom-global-fetch. This comment applies here, directly below (end of paragraph) and elsewhere if there are other instances.

Reply:
Done, I had to use `{{WindowOrWorkerGlobalScope/fetch()}}`.